dags/pull_here.py

```diff
@@ -86,23 +86,13 @@ def get_download_link(request_id: str, access_token: str):
     request_id =  get_request_id(access_token)
     download_url = get_download_link(request_id, access_token)
 
-    load_data_run = BashOperator(
-        task_id = "load_data",
-        bash_command = '''curl $DOWNLOAD_URL | gunzip | psql -h $HOST -U $USER -d bigdata -c "\COPY here.ta_view FROM STDIN WITH (FORMAT csv, HEADER TRUE);" ''', 
-        env = {"DOWNLOAD_URL": download_url, 
-               "HOST":  BaseHook.get_connection("here_bot").host, 
-               "USER" :  BaseHook.get_connection("here_bot").login, 
-               "PGPASSWORD": BaseHook.get_connection("here_bot").password},
-        append_env=True
-    )
-
-    #Can implement the following when we upgrade to 2.9.1 
-    #@task.bash(env={"DOWNLOAD_URL": download_url,
-    #                "HOST":  BaseHook.get_connection("here_bot").host,
-    #                "USER" :  BaseHook.get_connection("here_bot").login,
-    #           "        PGPASSWORD": BaseHook.get_connection("here_bot").password})
-    #def load_data_run()->str:
-    #    return '''curl $DOWNLOAD_URL | gunzip | psql -h $HOST -U $USER -d bigdata -c "\COPY here.ta_view FROM STDIN WITH (FORMAT csv, HEADER TRUE);" '''
+    @task.bash(env={"DOWNLOAD_URL": download_url,
+                    "HOST":  BaseHook.get_connection("here_bot").host,
+                    "USER" :  BaseHook.get_connection("here_bot").login,
+               "        PGPASSWORD": BaseHook.get_connection("here_bot").password})
```

Comment:
What's the deal with the big whitespace at the beginning of this line?

Reply:
:longcat_head: 
:longcat_torso:
:longcat_torso: 
:longcat_torso: 
:longcat_tail: 
You see nothing.